gpumon/gpumon.ml

```diff
@@ -1,7 +1,16 @@
+
+module type IMPLEMENTATION = sig
+  open Gpumon_interface
+
```

Comment:
Should this contain module Nvidis containing thr following functions? What will happen if we have to add a module Intel or a module A MD with similar functions?